### PR TITLE
Add Ubuntu reboot tests

### PR DESCRIPTION
**Pull request by amannch**
## What does this PR change?

Add reboot tests for Ubuntu minions.

## GUI diff

No difference.

- [x] **DONE**

## Documentation
- No documentation needed: Just tests

- [x] **DONE**

## Test coverage
- Cucumber Tests were added

- [x] **DONE**

### CODE REVIEW

**Comment by amannch** [comment]
Added small delay before following a link for Ubuntu and CentOS minions.
In my local setup the tests are not idempotent without it, because the second time the link gets generated too late and cucumber clicks on the link from the previous run

File: testsuite/features/allcli_reboot.feature [hunk 1]
```diff
@@ -47,22 +47,31 @@ Feature: Reboot systems managed by SUSE Manager
...
-  Scenario: Reboot the CentOS minion
+  Scenario: Reboot the CentOS minion and wait until reboot is completed
     Given I am on the Systems overview page of this "ceos-minion"
     When I follow first "Schedule System Reboot"
     Then I should see a "System Reboot Confirmation" text
     And I should see a "Reboot system" button
     When I click on "Reboot system"
     Then I should see a "Reboot scheduled for system" text
-
-@centos_minion
```
Comment (mateiw) [inline testsuite/features/allcli_reboot.feature]: I would keep the PR focused only for Ubuntu without touching other clients. Moreover, although the Ubuntu scenario looks simpler I'm not sure what was the initial reason for splitting reboot into two scenarios for the other clients. 
Let's add @Bischoff in the loop.

Comment (mateiw) [reply testsuite/features/allcli_reboot.feature]: Ups, sorry, missed the other comments.

**Comment by Bischoff** [comment]
> Thats why I rewrote the Ubuntu and CentOS tests

@amannch beware the CentOS tests are currently undergoing a big refactoring from @lkotek.

**Comment by amannch** [comment]
> @amannch beware the CentOS tests are currently undergoing a big refactoring from @lkotek.

Thanks for the heads-up. Right now these changes look like they are only for the core-features, so my changes here shouldn't be affected. I will, however, take a closer look at the changes and see if they would make sense for Ubuntu minions as well.

**Pull request merged**